elasticsearch/azuredeploy.json

```diff
@@ -295,23 +295,22 @@
     },
     "_artifactsLocation": {
       "type": "string",
-      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch/",
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch",
       "metadata": {
         "description": "Change this value to your repo name if deploying from a fork"
       }
     },
     "_artifactsLocationSasToken": {
       "type": "securestring",
-      "defaultValue": "?",
+      "defaultValue": "",
```

Comment:
I asked the same thing by was told that ? should be ok since there are no query parameters after it. If this is causing a problem then we need to change all existing templates where this change was made and also the guidance docs.

cc: @bmoore-msft